python/lsst/afw/geom/transform/transform.cc

```diff
@@ -51,10 +51,19 @@ std::string formatStr(Class const &self, std::string const &pyClassName) {
     return os.str();
 }
 
+template <class ExtraEndpoint, class FromEndpoint, class ToEndpoint, class PyClass>
+void declareOf(PyClass &cls) {
+    using PrefixClass = Transform<ExtraEndpoint, FromEndpoint>;
+    using CalledClass = Transform<FromEndpoint, ToEndpoint>;
```

Comment:
Is

    (FinalTransform (SecondTransform::*)(FirstTransform const &) const) & 
                     SecondTransform::template of<ExtraEndpoint>
clear?